packages/charts/src/renderers/xy-renderer.ts

```diff
@@ -1,8 +1,19 @@
 import { Renderer } from "../core/common/renderer";
 import { Scales } from "../core/common/scales/types";
-import { IAccessors, IDataSeries, IPosition } from "../core/common/types";
+import { IAccessors, IDataSeries, IPosition, IRendererEventPayload } from "../core/common/types";
+import { IRenderSeries } from "./types";
+import { Subject } from "rxjs";
+import isUndefined from "lodash/isUndefined";
+import { DATA_POINT_INTERACTION_RESET } from "../constants";
+import { UtilityService } from "../core/common/utility.service";
+import { IXYAccessors } from "./accessors/xy-accessors";
 
-export abstract class XYRenderer<TA extends IAccessors> extends Renderer<TA> {
+
+export class XYRenderer<TA extends IXYAccessors> extends Renderer<TA> {
+
+    public draw(renderSeries: IRenderSeries<TA>, rendererSubject: Subject<IRendererEventPayload>): void {
+
```

Comment:
Maybe a comment about why this is empty:

```suggestion
// This is empty to allow for series that represent metadata that may be shown in the legend but not visualized on the chart.
```